src/MenuDevConsole.cpp

```diff
@@ -233,6 +234,7 @@ void MenuDevConsole::execute() {
 		}
 		else {
 			log_history->add(msg->get("ERROR: Too few arguments"), false, &color_error);
+			log_history->add(msg->get("HINT: " + args[0] + " <category> [<x> <y>]"), false, &color_hint);
```

Comment:
`x` and `y` should be removed here, as `spawn_enemy` only supports specifying enemy category.